MiniApp/Classes/Display/MiniAppWebView.swift

```diff
@@ -1,2 +1,11 @@
-internal class MiniAppWebView {
+import WebKit
+
+internal class MiniAppWebView: WKWebView {
+
+    convenience init(filePath: URL) {
+        let config = WKWebViewConfiguration()
```

Comment:
Is JavaScript enabled by default, or does it need to be enabled manually? Maybe it's better to explicitly set it to enabled?

For Android WebViews, JS is disabled by default.

Reply:
Done